Example/Pellicola_Tests/DataStorageTests.swift

```diff
@@ -82,7 +82,9 @@ class DataStorageTests: XCTestCase {
             dataStorage.addImage($0, withIdentifier: UUID().uuidString)
         }
         
-        XCTAssertEqual(array, dataStorage.getImagesOrderedBySelection())
+        dataStorage.getImagesOrderedBySelection { urls in
+            XCTAssertEqual(array.count, urls.count)
```

Comment:
⚠️ This call is now asynchronous. Wait for the callback to be invoked with an expectation.